lib/inc/internal/facts/windows/uptime_resolver.hpp

```diff
@@ -30,7 +29,7 @@ namespace facter { namespace facts { namespace windows {
         virtual int64_t get_uptime() override;
 
      private:
-        std::shared_ptr<leatherman::windows::wmi> _wmi;
+        // std::shared_ptr<leatherman::windows::wmi> _wmi;
```

Comment:
Would rather remove this than comment it out.

Reply:
Yes...I actually wanted to get it thru one quick CI run (primarily to catch any compile issues) before deleting things.